packages/teams-js/src/artifactsForCDN/validDomains.json

```diff
@@ -34,6 +34,7 @@
     "teams.cloud.microsoft",
     "outlook.cloud.microsoft",
     "m365.cloud.microsoft",
-    "copilot.microsoft.com"
+    "copilot.microsoft.com",
+    "windows.msn.com"
```

Comment:
Are the domains in this list organized in any way? Should they be? 🤷‍♂️

Reply:
(also I'm assuming this change has already gotten security approval)

Reply:
Currently no to the organization, I think I can add some organization by grouping similar domain endpoints together. I've asked security to look at the domain and they were ok with adding windows.msn.com.